src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/GetAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,299 @@
+        [ValidateNotNullOrEmpty]
+        public string BackupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets whether or not to only get the latest backup per database.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ParameterSetName = ServerNameSet,
+            HelpMessage = "Whether or not to only get the latest backup per database. Defaults to false.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = DatabaseNameSet,
+            HelpMessage = "Whether or not to only get the latest backup per database. Defaults to false.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = LocationNameSet,
+            HelpMessage = "Whether or not to only get the latest backup per database. Defaults to false.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = GetBackupsByInputObjectSet,
+            HelpMessage = "Whether or not to only get the latest backup per database. Defaults to false.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = GetBackupsByResourceIdSet,
+            HelpMessage = "Whether or not to only get the latest backup per database. Defaults to false.")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter OnlyLatestPerDatabase { get; set; }
+
+        /// <summary>
+        /// Gets or sets the database state to look for (Alive or Deleted).
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ParameterSetName = ServerNameSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 4,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = DatabaseNameSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 4,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = LocationNameSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 4,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = GetBackupsByInputObjectSet,
+            Position = 2,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = GetBackupsByResourceIdSet,
+            Position = 2,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
```

Comment:
In swagger, the values are below. `Live` or `Alive` ?
```
            "enum": [
              "All",
              "Live",
              "Deleted"
            ],
```

Reply:
This comment was not addressed yet

Reply:
Alive. Sorry, missed this before.

Reply:
It's Alive, by the way.